source/options.html

```diff
@@ -4,65 +4,52 @@
+	<details id="token" open>
+		<summary><strong>🔑 Personal token</strong></summary>
+		<p>
+			Optional, <a id="personal-token-link" href="https://github.com/settings/tokens/new?description=Refined%20GitHub&scopes=repo,delete_repo">generate one</a><br>
 			<!-- placeholder is set to enable use of :placeholder-shown CSS selector -->
 			<input type="text" name="personalToken" pattern="[\da-f]{40}|ghp_\w{36,251}" placeholder=" " spellcheck="false" autocomplete="off" autocapitalize="off">
 			<span id="validation"></span>
-		</label>
-	</p>
-	<ul>
-		<li data-validation data-scope="valid_token">The token enables <a href="https://github.com/refined-github/refined-github/search?q=github-helpers+api">some features</a> to <strong>read</strong> data from public repositories
-		<li data-validation data-scope="public_repo">The <code>public_repo</code> scope lets them <strong>edit</strong> your public repositories
-		<li data-validation data-scope="repo">The <code>repo</code> scope lets them <strong>edit private</strong> repositories as well
-		<li data-validation data-scope="delete_repo">The <code>delete_repo</code> scope is only used by the <code>quick-repo-deletion</code> feature
-	</ul>
-
-	<hr>
-
-	<p>
-		<label>
-			<strong>🔗 Refined GitHub link</strong><br>
-			This page will open when clicking the toolbar icon
-			<input type="url" name="actionUrl" placeholder="https://github.com" spellcheck="false" autocomplete="off" autocapitalize="off">
-		</label>
-	</p>
-
-	<hr>
+		</p>
+		<ul>
+			<li data-validation data-scope="valid_token">The token enables <a href="https://github.com/refined-github/refined-github/search?q=github-helpers+api">some features</a> to <strong>read</strong> data from public repositories
+			<li data-validation data-scope="public_repo">The <code>public_repo</code> scope lets them <strong>edit</strong> your public repositories
+			<li data-validation data-scope="repo">The <code>repo</code> scope lets them <strong>edit private</strong> repositories as well
+			<li data-validation data-scope="delete_repo">The <code>delete_repo</code> scope is only used by the <code>quick-repo-deletion</code> feature
+		</ul>
+	</details>
 
-	<p>
-		<label>
-			<strong>💅 Custom CSS</strong> (useful to undo unwanted styles)
-			<textarea name="customCSS" rows="2" spellcheck="false"></textarea>
-		</label>
-	</p>
+	<details id="action">
+		<summary><strong>🔗 Refined GitHub link</strong></summary>
+		<p>This page will open when clicking the toolbar icon</p>
+		<p><input type="url" name="actionUrl" placeholder="https://github.com" spellcheck="false" autocomplete="off" autocapitalize="off"></p>
+	</details>
 
-	<hr>
+	<details id="css">
+		<summary><strong>💅 Custom CSS</strong></summary>
+		<p>Like a userstyle, useful to undo unwanted style changes</p>
+		<p><textarea name="customCSS" rows="2" spellcheck="false"></textarea></p>
+	</details>
 
-	<div>
-		<p id="show-debugging">
-			<button>Show debugging tools</button>
```

Comment:
The script and styles related to this `show-debugging` button can now also be removed.